### DISCUSSION TITLE
Handling Links

### DISCUSSION THREAD

**Discussion by konrain** [NuPlay/RichText#3] [Q&A] [answered]
Hey, thanks for the library!

How would you go about making sure links open in the safari app instead of opening inside the app?

**Comment by NuPlay** [comment]
If it's an app that supports iOS14 and above and you want to leave the app completely, then use Link.

**Reply by konrain** [reply]
@NuPlay what do you mean by that? maybe I should give an example

lets say I have an html like so:

```html
        <h1>Non quam nostram quidem, inquit Pomponius iocans;</h1>

here -> <a href="https://example.com">Example Link</a>

        <img src = "https://user-images.githubusercontent.com/img.png">
        
        <p>Lorem ipsum dolor sit amet <i>Quam illa ardentis</i> </p>


```

When the link inside Webview is clicked, it opens embedded within the app, we're looking to have it open in Safari

Thanks for your help

**Reply by NuPlay** [reply]
Oh, that's what you meant. I'll look for more about that.

**Comment by NuPlay** [comment, marked as answer]
| ![github1](https://user-images.githubusercontent.com/73557895/130707918-17e767c0-498e-4a37-8475-7aecac54639d.gif) 	| ![github2](https://user-images.githubusercontent.com/73557895/130707925-174d5715-07c3-4109-a3df-d2634552aa2e.gif) 	|
|-------------------------------------------------------------------------------------------------------------------	|-------------------------------------------------------------------------------------------------------------------	|
| Skip                                                                                                              	| Alert                                                                                                             	|

I'm going to add this option this weekend.

**Reply by konrain** [reply]
Looks perfect!
